packages/code-du-travail-frontend/src/outils/DureePreavisRetraite/steps/component/WarningResult.tsx

```diff
@@ -9,52 +9,57 @@ type Props = {
   ccNumber: number | null;
   type: "depart" | "mise" | null;
 };
-const titreFavorable = "Attention il peut exister une durée plus favorable";
+export const titreFavorable =
+  "Attention il peut exister une durée plus favorable";
 
-const titrePreavis =
+export const titrePreavis =
   "Attention il peut quand même exister une durée de préavis";
 
+export enum StatusCcn {
+  CCN_SUPPORTED = "CCN_SUPPORTED",
+  CCN_UNSUPPORTED = "CCN_UNSUPPORTED",
+  CC_UNSELECTED = "CC_UNSELECTED",
+}
+
 const WarningResult: React.FC<Props> = ({
   resultValueInDays,
   ccNumber,
   type,
 }) => {
-  const isSupported = React.useMemo(() => {
-    if (ccNumber) {
-      const idccInfo = supportedCcn.find((item) => item.idcc == ccNumber);
-      return !!(idccInfo && idccInfo.preavisRetraite);
-    }
-    return false;
+  const statusCcn: StatusCcn = React.useMemo(() => {
+    const idccInfo = supportedCcn.find((item) => item.idcc == ccNumber);
+    const find = !!(idccInfo && idccInfo.preavisRetraite);
```

Comment:
je trouve que find ça aide pas trop à comprendre le code

Reply:
Je suis pas sûr d'avoir compris. Perso, je trouve ça mieux qu'un `for`

Reply:
Ah désolé, j'avais pas compris, mais ok, oui je renomme la propriété :)